google/cloud/spanner/internal/connection_impl.cc

```diff
@@ -165,9 +196,11 @@ StatusOr<ResultSet> ConnectionImpl::ReadImpl(
   }
 
   auto context = google::cloud::internal::make_unique<grpc::ClientContext>();
-  auto rpc = stub_->StreamingRead(*context, request);
-  auto reader = internal::PartialResultSetSource::Create(std::move(context),
-                                                         std::move(rpc));
+  auto rpc =
+      google::cloud::internal::make_unique<DefaultPartialResultSetReader>(
+          std::move(context), stub_->StreamingRead(*context, request));
+
```

Comment:
Nit: Edit below doesn't add this blank line.

Reply:
Fixed.